docs/cpan-dependency-confusion.md

```diff
@@ -13,7 +13,6 @@ This type of vulnerability was proved to affect other ecosystems (PyPI, RubyGems
 In the Perl ecosystem, it could be one or more of the following options:
 - Using a "DarkPAN" along with CPAN
 - Using a "GrayPAN" along with CPAN
-- Using a "BackPAN" along with CPAN
```

Comment:
Carton (and I think cpanm) uses BackPAN when a module isn't available.

If a module is deleted from CPAN (possibly because it contains malware), it may still be on BackPAN.

Reply:
Yes, but I think that now BackPAN is more a "status" than a real distinct place.

For instance, https://cpan.metacpan.org/authors/id/S/SU/SUKRIA/ contains both CPAN and BackPAN releases. And MetaCPAN will return both.

I'm not sure we can confuse using BackPAN... Can we?